packages/mrm-task-eslint/index.js

```diff
@@ -67,15 +72,24 @@ module.exports = function task({
 
 	// .eslintrc.json
 	const eslintrc = json(configFile, legacyConfig);
-	const hasCustomPreset = _.castArray(eslintrc.get('extends', [])).find(x =>
-		x.startsWith(eslintPreset)
-	);
+	const extnds = _.castArray(eslintrc.get('extends', []));
```

Comment:
```suggestion
	const extends = _.castArray(eslintrc.get('extends', []));
```

Reply:
Unfortunately `extends` is a reserved word.

Reply:
It's not an excuse for a typo. How about `extendsValue`?

Reply:
Sure, done.